apex/cli/setup.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from setuptools import setup
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+requirements = [
+    # TODO: put package requirements here
```

Comment:
Oops, yeah, this was just some boilerplate that I got from a cookiecutter template.